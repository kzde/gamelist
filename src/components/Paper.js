import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';
import { colorPalettes, themeColors } from '../utils/styleUtils';

const Wrapper = styled.div`
  background: ${colorPalettes.white};
  border: 1px solid ${themeColors.borderGrey};
`;

const Paper = ({children}) => (
  <Wrapper>
    {children}
  </Wrapper>
)

Paper.propTypes = {
  children: PropTypes.element.isRequired
};
export default Paper;