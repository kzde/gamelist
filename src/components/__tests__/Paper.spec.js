import React from 'react';
import renderer from 'react-test-renderer';
import Paper from '../Paper';

describe('Paper', () => {
  it('should renders correctly', () => {
    const tree = renderer
      .create(<Paper children={<div>paper</div>} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
})