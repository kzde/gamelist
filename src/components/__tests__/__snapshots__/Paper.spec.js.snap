// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper should renders correctly 1`] = `
<div
  className="Paper__Wrapper-xda1y4-0 ieAraY"
>
  <div>
    paper
  </div>
</div>
`;
